kong/api/endpoints.lua

```diff
@@ -1,5 +1,5 @@
 local Errors       = require "kong.db.errors"
-local utils        = require "kong.tools.utils"
+local utils        = require "kong.tools.uuid"
```

Comment:
Should we update the local variable name at the same time? such as `utils` -> `uuid`.

Reply:
Sure, it will be improve readability, but it will also increase the complexity of this PR.
I will leave it to another PR, perhaps you could do it in the future.

Reply:
I don't understand why we want to skip making this change? Definitely not call `uuid` as `utils`. What is the point?

Reply:
Got, I will rename these modules.